src/pages/showcase.js

```diff
@@ -21,116 +21,160 @@ const projects = [
+    }),
     image: imgPath + 'debug-trainer/logo.png',
     link: 'https://github.com/kickstartcoding/debug_trainer_app',
     screenshots: ['0.png'].map((image) => imgPath + 'debug-trainer/' + image),
   },
   {
     title: 'Authme Lite',
-    description: translate({message: "A lite and simple two-factor authentication app for desktop."}),
+    description: translate({
+      message: 'A lite and simple two-factor authentication app for desktop.',
+    }),
     image: imgPath + 'authme-lite/logo.png',
     link: 'https://github.com/Levminer/authme-lite',
     screenshots: ['0.png'].map((image) => imgPath + 'authme-lite/' + image),
   },
   {
     title: 'Mail-Dev',
-    description: translate({message: "Local SMTP Server For Email Testing/Debugging."}),
+    description: translate({
+      message: 'Local SMTP Server For Email Testing/Debugging.',
+    }),
     image: imgPath + 'mail-dev/logo.png',
     link: 'https://github.com/samirdjelal/mail-dev/',
     screenshots: ['0.png'].map((image) => imgPath + 'mail-dev/' + image),
   },
   {
     title: 'Commandos',
-    description: translate({message: "Open Source Git Client."}),
+    description: translate({ message: 'Open Source Git Client.' }),
     image: imgPath + 'commandos/logo.png',
     link: 'https://github.com/Commandos-app/Commandos/',
     screenshots: ['0.png'].map((image) => imgPath + 'commandos/' + image),
   },
   {
     title: 'ExcelMerger',
-    description: translate({message: "ExcelMerger merges Excel files"}),
+    description: translate({ message: 'ExcelMerger merges Excel files' }),
     image: imgPath + 'excelmerger/logo.png',
     link: 'https://github.com/Leon0824/ExcelMerger',
     screenshots: ['0.png'].map((image) => imgPath + 'excelmerger/' + image),
   },
   {
     title: 'Tinker 2',
-    description: translate({message: "REPL desktop app tinker with your PHP applications without touching source code."}),
+    description: translate({
+      message:
+        'REPL desktop app tinker with your PHP applications without touching source code.',
+    }),
     image: imgPath + 'tinker2/logo.png',
     link: 'https://github.com/bangnokia/tinker2/',
     screenshots: ['0.png'].map((image) => imgPath + 'tinker2/' + image),
   },
   {
     title: 'Xplorer',
-    description: translate({message: "A customizable and modern file manager"}),
+    description: translate({
+      message: 'A customizable and modern file manager',
+    }),
     image: imgPath + 'xplorer/logo.png',
     link: 'https://github.com/kimlimjustin/xplorer/',
     screenshots: ['0.png'].map((image) => imgPath + 'xplorer/' + image),
   },
   {
     title: 'Kadium',
-    description: translate({message: "App for staying ontop of YouTube channel uploads."}),
+    description: translate({
+      message: 'App for staying ontop of YouTube channel uploads.',
+    }),
     image: imgPath + 'kadium/logo.svg',
     link: 'https://github.com/samirdjelal/mail-dev/',
```

Comment:
Can you fix the Kadium link while you're at it?

Reply:
Yes, fixed